shared/settings/nav/settings-item.tsx

```diff
@@ -26,6 +26,7 @@ export default function SettingsItem(props: SettingsItemProps) {
               borderLeftColor: Styles.globalColors.blue,
               borderLeftStyle: 'solid',
               borderLeftWidth: 3,
+              ...(Styles.isTablet ? {borderRadius: 0} : {}),
```

Comment:
might be cleaner to move these styles into the `styles` object and use `platformStyles` for the tablet-specific stuff